src/main/java/org/frc5687/rapidreact/RobotContainer.java

```diff
@@ -67,6 +68,7 @@ public void init() {
         _oi.initializeButtons(_driveTrain, _catapult, _intake, _climber);
         _robot.addPeriodic(this::controllerPeriodic, 0.005, 0.005);
         _imu.reset();
+        _oi.initializeButtons(_driveTrain);
```

Comment:
You've got 2 calls to oi.initializeButtons with different parameters

Reply:
Thanks for catching that. Fixed.